web/viewer.css

```diff
@@ -1844,6 +1844,10 @@ html[dir="rtl"] #documentPropertiesOverlay .row > * {
     align-items: center;
   }
 
+  #printContainer .xfaPage {
+    position: initial;
+  }
+
```

Comment:
Looking at the canvas-related printing rules further down, do you perhaps want to do something like `#printContainer > .xfaPrintedPage .xfaPage` instead here (e.g. for consistency)?